app/src/main/java/my/noveldokusha/scraper/WuxiaWorld.kt

```diff
@@ -0,0 +1,129 @@
+package my.noveldokusha.scraper.sources
+
+import my.noveldokusha.data.BookMetadata
+import my.noveldokusha.data.ChapterMetadata
+import my.noveldokusha.scraper.*
+import org.jsoup.nodes.Document
+
+class WuxiaWorld : SourceInterface.catalog
+{
+    override val name = "WuxiaWorld"
+    override val baseUrl = "https://wuxiaworld.site/"
+    override val catalogUrl = "https://wuxiaworld.site/novel/?m_orderby=views"
+    override val language = "English"
+
+    override suspend fun getChapterTitle(doc: Document): String? = null
+
+//    override suspend fun getChapterText(doc: Document): String
+//    {
+//        return doc.selectFirst(".reading-content p")!!.let { textExtractor.get(it) }
+//    }
+
+    override suspend fun getBookCoverImageUrl(doc: Document): String?
+    {
+        return doc.selectFirst("div.summary_image")
+            ?.selectFirst("img")
+            ?.attr("data-src")
```

Comment:
need to change the base curl it get from https://wuxiaworld.b-cdn.net/ to https://wuxiaworld.site/